pkgs/os-specific/darwin/apple-sdk/frameworks.nix

```diff
@@ -82,9 +82,7 @@ with frameworks; with libs; {
   PCSC                    = [ CoreData ];
   PreferencePanes         = [];
   PubSub                  = [];
-  QTKit                   = [ CoreMediaIO CoreMedia MediaToolbox QuickTime VideoToolbox ];
```

Comment:
Why did you remove a bunch of these? They are still in the SDK.

Reply:
Are you sure? Maybe they’re in another download. I’ll find the the error i was getting...

Reply:
I looked at `darwin.apple_sdk.sdk` which is the source IIRC.

Reply:
Maybe just quicktime was removed?

Reply:
Ah indeed, the QuickTime headers are no longer there.